code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -329,7 +329,7 @@
 	desc = "A tough, hard covering of black chitin with transparent chitin in front."
 	icon_state = "lingarmorhelmet"
 	flags = BLOCKHAIR | NODROP
-	armor = list(melee = 40, bullet = 15, laser = 30,energy = 10, bomb = 5, bio = 2, rad = 0)
+	armor = list(melee = 60, bullet = 30, laser = 60, energy = 30, bomb = 10, bio = 4, rad = 0)

```

Comment:
Helmets always have less damage resist than the main body armor. See: security helmets vs security vests